Support for reportsv2.php

Fl0w3D opened the pull request:
Adds support for dynamic forms in reportsv2.php (Gazelle).
Kind of preliminary work, it must be perfectible (like removing the interval only if the innerHTML of the form has actually changed).

Fl0w3D committed:
Support for reportsv2.php

Adds support for dynamic forms in reportsv2.php (Gazelle).
Kind of preliminary work, it must be perfectible (like removing the interval only if the innerHTML of the form has actually changed).

hateradio commented:
Two things:

1. Could you move the `if (document.getElementById('type')) {` block into the `if (document.getElementById('content')) {` block? That's intended to be for Gazelle only.

2. In the `setInterval` function I had added a check to see if the textarea had been loaded. If it didn't the function emits again until it appears. Can you add a `if (txt) {` condition like the one in `edit: function (evt) {`. If emitting multiple times until a textarea loads is not required, change `setInterval` to `setTimeout` and remove `interv`.

Fl0w3D committed:
Update whutbbcode.user.js

#type change bound only if #content exists (Gazelle), replaced setInterval by setTimeout in report: event.

Fl0w3D committed:
Update whutbbcode.user.js

Fl0w3D committed:
Update whutbbcode.user.js

Yet another replacement of setInterval with setTimeout

Fl0w3D committed:
Update whutbbcode.user.js

Remove interv declaration since not used anymore.

Fl0w3D commented:
Here.
